tools/nodejs/test/interrupt.test.ts

```diff
@@ -1,13 +1,13 @@
-var sqlite3 = require('..');
-var assert = require('assert');
+import * as sqlite3 from '..';
+import * as assert from 'assert';
+import {DuckDbError} from "..";
 
 // FIXME each is not streaming yet
-return
 
-describe('interrupt', function() {
+describe.skip('interrupt', function() {
```

Comment:
Similar comment to earlier. Do we have to skip?  Running at least exercises the code path and ensures it is not crashing. (Obviously adding some kind of assertion would be preferable though).

Reply:
These tests weren't running at all previously, I'll see what's required to get them passing

Reply:
Again, sorry I missed if this was commented out!  No need to get it running in that case; best to just get this PR landed.